lib/connect.js

```diff
@@ -304,12 +304,13 @@ function connect(token, options) {
  *   for the {@link LocalParticipant} to {@link NetworkQualityVerbosity}<code style="padding:0 0">#minimal</code>
  *   and the {@link NetworkQualityVerbosity} for {@link RemoteParticipant}s to
  *   {@link NetworkQualityVerbosity}<code style="padding:0 0">#none</code>.
- * @property {string} [region='gll'] - Preferred region; By default, the client will connect to the
+ * @property {string} [region='gll'] - Preferred signaling region; By default, you will be connected to the
  *   nearest signaling server determined by latency based routing. Setting a value other
  *   than <code style="padding:0 0">gll</code> bypasses routing and guarantees that signaling traffic will be
- *   terminated in the region that you prefer. If you are connecting to a Group Room created with the <code style="padding:0 0">gll</code>
+ *   terminated in the region that you prefer. If you are the first to join a Group Room created with the <code style="padding:0 0">gll</code>
  *   Media Region (either Ad-Hoc or via the REST API), then the Room's Media Region will be selected
- *   based upon your client's region
+ *   based upon your signaling region. Please refer to this <a href="https://www.twilio.com/docs/video/ip-address-whitelisting#signaling-communication" target="_blank">table</a>
+ *   for the list of supported signaling regions.
```

Comment:
I would remove the sentence about GLL media behavior until we are ready with the feature. This would match the iOS docs.

Other than that it looks good to me. I think we should take another pass at the IDL to arrive on final terminology before GA.

Reply:
Although I prefer names like "auto" or "global" we decided to converge on the same API term that Voice uses in their APIs for consistency across Voice and Video, which is "gll" (not my favorite, but we felt consistency was more useful to customers than a new, better name...)

Reply:
If you asked me 2 months ago I would say let's use "gll" for consistency :) However, now I would not consider this API consistency to be a factor in deciding on the right API given how far apart Voice and Video are.  Everyone Ive spoken to internally has said gll is not a good name. We have an opportunity to give it a better name! Again, this would only be public facing. Internally, we can call it whatever you like.